### DISCUSSION TITLE
How to run multiple particles simultaneously from the same location

### DISCUSSION THREAD

**Discussion by RomanIMMSP** [Parcels-code/Parcels#1411] [Q&A] [answered]
Dear Parcels community

Maybe this question very simple, but I did't find the answer in the documentation.
I would like to run multiple particles (for example, 1000) simultaneously from the same location to discover the impact of diffusion.
How to do this in the easiest way?

This is the standard Parcels code
```
lon = XX
lat = YY
dep = DD
pset = ParticleSet(fieldset=fieldset, pclass=JITParticle, lon=lon, lat=lat, depth=dep)
```

In such case only one particle will release. How to multiply it by 1000 times?

Thanks for the answer in advance.

**Comment by michaeldenes** [comment, marked as answer]
HI @RomanIMMSP,

A simple way to do this would be to multiply your constants by `np.ones(1000)` so that you are passing in arrays, e.g.

`lon = XX * np.ones(1000)`
`lat = YY*np.ones(1000)`
`dep = DD*np.ones(1000)`
`pset = ParticleSet.from_list(fieldset=fieldset, pclass=JITParticle, lon=lon, lat=lat, depth=dep)`

Let me know how you go! Cheers,
Michael

**Reply by RomanIMMSP** [reply]
Thank you very much.
I applied this and had an error
'float' object has no attribute 'ones'

**Reply by michaeldenes** [reply]
My apologies, it should read
`depths=dep`
Let me know if that works!

**Comment by RomanIMMSP** [comment]
Hi Michael

Thanks for your hint. It works.
Actually, your first code is right. The problem was in my code.

Best wishes,
Roman

**Reply by michaeldenes** [reply]
No problem, best of luck with it all!
